### PR TITLE
Fix casing in Slack and GitHub

### PR DESCRIPTION
**Pull request by NMinhNguyen**
I didn't like how GitHub was misspelt on the main page so thought I'd contribute this little fix. Whilst looking for `Github`, I also spotted Slack in lowercase and fixed it too.
 
## Before

image

## After

image

If you decide to accept this PR and it's not too much of an ask, could you label the PR with `hacktoberfest-accepted`?

### CODE REVIEW

File: docs/passportjs.mdx [hunk 1]
```diff
@@ -58,7 +58,9 @@ export default passportAuth({
   successRedirectUrl: "/",
   errorRedirectUrl: "/",
   secureProxy: true, // highlight-line
-  strategies: [/*...*/],
+  strategies: [
```
Comment (NMinhNguyen) [inline docs/passportjs.mdx:61]: Prettier did this, happy to revert

File: docs/maintainers.mdx [hunk 2]
```diff
@@ -55,8 +55,7 @@ Some especially important points:
 - **Gratitude:** immediately express gratitude when someone opens an issue or PR. This takes effort/time and we appreciate it
 - **Responsiveness:** during issue/PR triage, even if we can’t do a full review right away, leave a comment thanking them and saying we’ll review it soon
 - **Understanding:** it's critical to ensure you understand exactly what someone is saying before you respond. Ask plenty of questions if needed. It's very bad if someone has to reply to your response and say "actually I was asking about X"
-  - In fact, at least one question is almost always required before you can respond appropriately  — whether in Github or in Slack
-
```
Comment (NMinhNguyen) [inline docs/maintainers.mdx:59]: Prettier did this too

File: docs/contributing.mdx [hunk 3]
```diff
@@ -34,7 +34,7 @@ The Blitzjs.com website and documentation repo also has issues with [`ready to w
 - Performance or security improvements
 - Educational content like blogs, videos, courses
 
-If there's some other way you'd like to contribute, ask us about it in slack!
+If there's some other way you'd like to contribute, ask us about it in Slack!
```
Comment (NMinhNguyen) [inline docs/contributing.mdx:37]: I suspect it's actually `on Slack` and `on GitHub`  but not 100% confident.

**Comment by flybayer** [comment]
Thank you :)

**Comment by NMinhNguyen** [comment]
Thanks @flybayer! Would you mind adding a `hacktoberfest-accepted` label?

**Pull request merged**